doc/manual/src/glossary.md

```diff
@@ -11,6 +11,32 @@
     The location in the file system where store objects live. Typically
     `/nix/store`.
 
+    From   the  perspective   of   the  location   where  Nix   is
+    invoked<sup><b>1</b></sup>, the  Nix store can be  referred to
+    as a "_local_" or a "_remote_" one:
+
+    <sup>\[1]: Where "invoking Nix" means  an executing a Nix core
+    action/operation on  a Nix store.  For example, using  any CLI
+    commands from the `NixOS/nix` implementation.</sup>
```

Comment:
Any reason we cannot use markdown footnotes (`[^1]` and `[^1]: ...`)? If no, please make sure the number is not otherwise used.

Other than that, I doubt the footnote helps. There is no such thing as a "Nix core action/operation", and if we want to reference the CLI commands, let us link the relevant manual section.

This also confuses me: Is there any other way to "invoke Nix" other than through commands? Sure, if you somehow call `libstore`, but does that really matter here? I think for the sake of getting the message across, invoking Nix should be enough.
```suggestion
    invoked, the  Nix store can be  referred to
    as a "_local_" or a "_remote_" one:
```